### DISCUSSION TITLE
Aggregator train accuracy

### DISCUSSION THREAD

**Discussion by MuhammadAshiqAmeer** [securefederatedai/openfederatedlearning#735] [Q&A] [answered]
As aggregator is not having any data, I am confused how the aggregator train result train_acc ,locally_tuned_model_validate result validation_accuracy,aggregated_model_validate result validation_accuracy are getting. Does aggregator have any access to validation set of collaborators?

**Comment by psfoley** [comment, marked as answer]
Hi Muhammad!

You're right that the aggregator does not have any data. The accuracy reported on the aggregator is the weighted average of the collaborator metrics that are reported after the round is complete. It's intended to give a representation of how much the combined model is improving on the collaborator's local data round over round.

**Reply by Jinal4502** [reply]
hey, is there any way to perform validation on aggregator side after every round?
